charts/k8ssandra-cluster/templates/grafana/datasource.yaml

```diff
@@ -12,6 +12,6 @@ spec:
         timeInterval: 5s
       name: Prometheus
       type: prometheus
-      url: http://{{ .Release.Name }}-prometheus-k8ssandra.{{ .Release.Namespace }}:9090
+      url: http://{{ .Release.Name }}-prometheus-k8ssandra.{{ .Release.Namespace }}:9090/{{ .Values.monitoring.prometheus.routePrefix }}
```

Comment:
I think we want a check here to see if `routePrefix` is defined.

Reply:
Updated with test coverage.